# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/WorkletFetchHandler.cpp
# Commit: 75ae330e1b90
# Full Hash: 75ae330e1b90273c1eafb42b21098b3753e4954a
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-24 18:03 +0000
# Regressor Bug: 1968890
# File Overlap Count: 3
# Overlapping Files: dom/worklet/WorkletFetchHandler.cpp, dom/workers/ScriptLoader.cpp, dom/script/ScriptLoader.cpp
# Description:
#   Bug 1968890 - Part 6: ScriptLoaders update. r=jonco,dom-worker-reviewers,edenchuang
# ==============================================================================

diff -r c48ad530cfe2 -r 75ae330e1b90 dom/worklet/WorkletFetchHandler.cpp
--- a/dom/worklet/WorkletFetchHandler.cpp	Wed Sep 24 18:03:30 2025 +0000
+++ b/dom/worklet/WorkletFetchHandler.cpp	Wed Sep 24 18:03:30 2025 +0000
@@ -220,15 +220,6 @@
 
   request->mBaseURL = mURI;
   request->OnFetchComplete(mResult);
-
-  if (NS_FAILED(mResult)) {
-    if (request->IsTopLevel()) {
-      request->LoadFailed();
-    } else {
-      request->Cancel();
-    }
-  }
-
   moduleLoader->RemoveRequest(mURI);
   return NS_OK;
 }
